content/blog/2019/05/2019-05-30-becoming-contributor-newbie-tickets.adoc

```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: "The journey of becoming a Jenkins contributor: Newbie-friendly tickets"
+tags:
+- jenkins
+- community
+- developer
+- contributing
+- newcomer
+author: romenrg
```

Comment:
You can improve social media references to this blog post by adding a few more entries here.  Refer to the new content in "[Adding a blog post](https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post)" for more details.

```suggestion
author: romenrg
description: "Fixing a newbie friendly Jenkins issue"
opengraph:
  image: /images/post-images/2019-05-30-becoming-contributor/pr-merged.png
```